### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by milen-ale**
This is a simple todo list web page built using Webpack and served by a Webpack dev server. It features the CRUD operations.
- Data is stored in localStorage
-. Has some automated tests written using Jest.

### CODE REVIEW

**Review by codepantha — APPROVED** [review]
# STATUS: APPROVED 🟢 🍷 

## Hi team 👋, 

You have done an amazing job 👏 
Your project is now complete, and there is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### Highlights ⭐ 
✔️ Descriptive PR
✔️ Tests have been written for adding and deleting to-do items
✔️ Good commit history
✔️ No linter errors

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/crudOps.test.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+/* eslint-disable */
```
Comment (arslanbisharat) [inline src/crudOps.test.js:1]: - [x] Please make sure all the linter rules are enabled. These are here to help us write clean code.

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
+/* eslint-disable */
```
Comment (arslanbisharat) [inline src/index.js]: - [x] Please make sure all the linter rules are enabled. These are here to help us write clean code.

File: webpack.config.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
+/* eslint-disable */
```
Comment (arslanbisharat) [inline webpack.config.js]: - [x] Please make sure all the linter rules are enabled. These are here to help us write clean code.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
Hi @milen-ale, @codepantha, and @MxoDevops, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- The Pull Request has a proper title and description. ✔️
- You have added a descriptive Readme file. Good Job. ✔️
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [x] Please make sure you right descriptive comments. Commenting involves placing Human Readable Descriptions inside of computer programs detailing what the Code is doing. Proper use of commenting can make code maintenance much easier, as well as helping make finding bugs faster. Further, commenting is very important when writing functions that other people will use. :x:


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by KDlamini — APPROVED** [review]
### Hi team @milen-ale, @MxoDevops 🙋‍♀️

Excellent work! 💯 
You have done a great job implementing all the required changes. 
Your feature is complete! There is nothing else to say other than... it's time to merge it. :shipit: 
Congratulations!!!🎊 🥳 

## Highlights 🌞 
✔️  Add and remove methods are tested
✔️  No linter errors

# Status: Approved ✔️ 

giphyapproved
 
Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**